### PR TITLE
Refactor aave history to stats

### PR DESCRIPTION
**Pull request by tewshi**
## Checklist

- [x] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/defi/Borrowing.vue [hunk 1]
```diff
@@ -44,15 +44,15 @@ const refresh = async () => {
   await defiLending.fetchBorrowing(true);
 };
 
-onMounted(async () => {
+onMounted(() => {
```
Comment (kelsos) [inline frontend/app/src/components/defi/Borrowing.vue:47]: Is there any reason we are removing the async/await here?

File: frontend/app/src/components/defi/Borrowing.vue [hunk 2]
```diff
@@ -44,15 +44,15 @@ const refresh = async () => {
...
   const currentRoute = get(route);
   const queryElement = currentRoute.query['protocol'];
   const protocols = Object.values(DefiProtocol);
   const protocolIndex = protocols.indexOf(queryElement as DefiProtocol);
   if (protocolIndex >= 0) {
-    set(protocol, protocols[protocolIndex]);
+    set(protocol, protocols[protocolIndex] as DefiProtocol);
```
Comment (kelsos) [inline frontend/app/src/components/defi/Borrowing.vue:53]: I am curious, did this work fine without the cast before?

File: frontend/app/src/composables/defi/lending.ts [hunk 3]
```diff
@@ -2,10 +2,8 @@ import { type Balance, BigNumber } from '@rotki/common';
 import { DefiProtocol } from '@rotki/common/lib/blockchain';
 import { assetSymbolToIdentifierMap } from '@rotki/common/lib/data';
 import {
-  AaveBorrowingEventType,
   type AaveHistoryTotal,
-  type AaveLending,
-  isAaveLiquidationEvent
+  type AaveLending
```
Comment (kelsos) [inline frontend/app/src/composables/defi/lending.ts:6]: Can you check if the `isAaveLiquidationEvent` is used anywhere else? if not we should remove it.

**Review by kelsos — COMMENTED** [review]
I have left a couple of questions, but I tested and it looks ok

File: frontend/app/src/components/defi/Borrowing.vue (hunk 1 above)
Comment (tewshi) [reply frontend/app/src/components/defi/Borrowing.vue:47]: it is causing i18n to break...

File: frontend/app/src/composables/defi/lending.ts (hunk 3 above)
Comment (tewshi) [reply frontend/app/src/composables/defi/lending.ts:6]: ok

**Review by kelsos — APPROVED** [review]
lgtm, thank you

**Pull request merged**